pages/linux/mcookie.md

```diff
@@ -0,0 +1,19 @@
+# mcookie
+
+> Generates random 128 bit hexadecimal numbers.
+
+- Generate a random number:
+
+`mcookie`
+
+- Generate a random number, using the contents of a file as a seed for the randomness:
```

Comment:
I think we can just use one file in these examples, to keep it simple.

Reply:
I quite not sure about this. Let us have the third opinion please. @sbrl

Reply:
Well having 2 files surely explains better what the command does. But the whole idea behind tldr is to exemplify in the most simple way the most common form of use of the commands we maintain.

I would prefer to have only one file on the example as more than one could cause some confusion.

Reply:
Ok, I'll change it to have just only one file. Thank you for your comments! @agnivade @schneiderl

Reply:
Hmmmm. I think, in this instance, if someone wants to specify multiple files, it should be fairly intuitive to use `--file` multiple times. To this end, I think it's ok if we don't show this off specifically here.

It sounds like an advanced use-case, which generally isn't a good fit for a tldr page. If it's something that users of this command would need regularly, then we should probably reconsider.